backend/views/base.py

```diff
@@ -287,28 +295,53 @@ def register_user(request):
     )
 
 
-# Login existing user
 def login_user(request):
     if request.method == "POST":
         username = request.POST["username"]
         password = request.POST["password"]
         remember_me = request.POST.get("remember_me")
-        user = authenticate(request, username=username, password=password)
-        if user is not None:
-            login(request, user)
-            if remember_me:
-                request.session.set_expiry(604800)
+
+        try:
+            user = User.objects.get(username=username)
+        except User.DoesNotExist:

```

Comment:
[IMHO] Do you know this method?
`User.objects.filter(username=username).exists()`